src/components/Header/index.tsx

```diff
@@ -383,7 +383,7 @@ function Header() {
             {isUnsupportedChainIdError ? (
```

Comment:
I think you can reduce this code to something like that
![image](https://user-images.githubusercontent.com/46563377/160149142-37a92804-eee0-4c19-8b47-2591896299b4.png)